Source/Blazorise/bundleconfig.json

```diff
@@ -0,0 +1,15 @@
+[
+  {
+    "outputFileName": "wwwroot/blazorise.js",
+    "inputFiles": [
+      "Scripts/blazorise.dependencies.js",
```

Comment:
No need for `blazorise.` prefix for source files

Reply:
Think this config also supports wildcard. So something like Scripts/*.js